translate/storage/test_mo.py

```diff
@@ -156,17 +155,14 @@ def test_output(self):
                 # can skip the checks here.
                 continue
 
-            mo_msgfmt_f = open(MO_MSGFMT)
-            mo_pocompile_f = open(MO_POCOMPILE)
-
-            try:
+            with open(MO_MSGFMT, 'rb') as mo_msgfmt_f:
                 mo_msgfmt = mo_msgfmt_f.read()
-                print("msgfmt output:")
-                print(repr(mo_msgfmt))
+            print("msgfmt output:")
+            print(repr(mo_msgfmt))
```

Comment:
Sorry, you are right. Somehow I was mixing different concepts on my head. My apologies.